Server.Core/Response.cs

```diff
@@ -6,7 +6,13 @@ public class Response
     {
         public int StatusCode { get; set; }
         public string ContentType { get; set; }
-        public MemoryStream Body { get; set; }
+        public Stream Body { get; set; }
+
+        public void SetBody(byte[] bodyMessage)
```

Comment:
Hmm... you are right, but I am not sure which is wrong at the moment.  I'll have to think about that one.